custom_components/tuya_local/devices/trotec_pac-w_heatpump.yaml

```diff
@@ -70,11 +70,20 @@ primary_entity:
       name: countdown
       type: integer
     - id: 106
-      name: alarm
+      name: fault_code
       type: integer
```

Comment:
fault_codes are probably bitfield types. (Tuya calls them bitmap, but I used bitfield to avoid confusion with the image format).  Without a mapping there isn't any difference between integer and bitfield, but it might help future proof the config.